test/unit/db/query-parsers/sql.test.js

```diff
@@ -158,7 +158,7 @@ test('database query parser', async (t) => {
           t.todo('should strip database names from collection names as ' + cat.table)
         } else {
           await t.test('should parse the collection as ' + cat.table, function () {
```

Comment:
can you remove this sub test, it's not necessary, my suggestion got messed up it appears